package.json

```diff
@@ -14,11 +14,18 @@
         "build": "microbundle --jsx React.createElement",
         "dev": "microbundle watch --jsx React.createElement"
     },
+    "files": [
+        "src",
+        "dist"
+    ],
     "dependencies": {
-        "auth0-js": "^9.11.2",
-        "react": "^16.8.6"
+        "auth0-js": "^9.11.2"
     },
     "devDependencies": {
-        "microbundle": "^0.11.0"
+        "microbundle": "^0.11.0",
+        "react": "^16.8.6"
+    },
+    "peerDependencies": {
+        "react": "^16.7.0"
```

Comment:
You probably want this to say `^16.8.0` as that’s the one that introduced Hooks, not 16.7.0.

Reply:
sgtm - I just guessed at the version.